Snakemake/workflow/rules/infer_trees.smk

```diff
@@ -8,21 +8,24 @@ rule prepare_sample_file:
     input:
         vcf=f'{vcfdir}/{{chromosome}}_ancestral.vcf',
         meta=config['meta']
-    conda:
-        "tsinfer"
     output:
         "../Project/Tsinfer/samples/{chromosome}.samples"
     params:
         chrLength= lambda wildcards:  config['chromosome_length'][int(re.findall(r'\d+', wildcards.chromosome)[0])],
         ploidy=config['ploidy']
+    conda: "HLab_tsinfer"
```

Comment:
So here you are using conda env? why are you using modules for the other ones then?